Support Vector Machine/SVM_Linear_Kernal_&_documentation.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import pandas as pd
+
+
+# The dataset is from a kaggle competition https://www.kaggle.com/c/titanic The main aim is to find whether a
+# passenger of a certain sort is most likely to survive based on features like name, age, gender, socio-economic
+# classes, etc)
+#class SVM is implementing Support Vector Machine from Scratch
+class SVM:
+
+    def __init__(self, alpha=0.001, lambda1=0.01, epochs=1000):
+        self.alpha = alpha
+        self.lambda1 = lambda1
+        self.epochs = epochs
+        self.weights = None
+        self.b = None
+
+    def fit(self, X, y):
+        cols, rows = X.shape
+        y1 = np.where(y <= 0, -1, 1)
+        self.weights = np.random.randn(rows)
+        self.b = 0
+
+        for _ in range(self.epochs):
+            for i in range(len(y1)):
+                if y1[i] * (np.dot(X[i], self.weights) - self.b) >= 1:
+                    self.weights -= self.alpha * (2 * self.lambda1 * self.weights)
+                else:
+                    self.weights -= self.alpha * (2 * self.lambda1 * self.weights - y1[i] * X[i])
+                    self.b -= self.alpha * y1[i]
+
+    def predict(self, X):
+        predict_ = np.dot(X, self.weights) - self.b
+        for i in range(len(predict_)):
+            if predict_[i] == -1:
+                predict_[i] = 0
+        return np.sign(predict_)
+
+#Helps in calculating model accuracy
+def model_accuracy(y_test, pred):
+    global acc
+    sum = 0
+    for i in range(len(prediction)):
+        if y_test[i] == prediction[i]:
+            sum = sum + 1
+            acc = sum / len(prediction)
+
+    return acc
+
+
+def initialization():
```

Comment:
@ptnv-s after this,the content is redundant

Reply:
please comment it out

Reply:
@ashwani-rathee  did it in latest commit ,could you review it please